lib/libc/string/wcscmp.c

```diff
@@ -48,9 +48,11 @@ int
 wcscmp(const wchar_t *s1, const wchar_t *s2)
 {
 
-	while (*s1 == *s2++)
-		if (*s1++ == '\0')
+	while (*s1 == *s2) {
+		if (*s1 == L'\0')
 			return (0);
+		++s1, ++s2;
+	}
```

Comment:
So what's the benefit from making this more complex?

Reply:
More complex is doing the ++ and then the — in the return statement, not this.

Reply:
this change is churn, I'm not going to risk merging it.